package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "oref0",
-  "version": "0.7.0",
+  "version": "0.7.1",
```

Comment:
I think this is an artifact? Will need to check it before merging.

Reply:
I'm not sure! All I can say is that it wasn't in my original changeset.

Reply:
It was because you based your change on master instead of dev, so shows up as a diff after I rebased to dev.

Reply:
Ah, of course. My bad on the base branch. Fortunately, it doesn't seem to make a significant difference in this context. Let me know if you would like me to do anything on my end.

Reply:
I'll probably `git reset --hard origin/dev` and cherry-pick your commits, then git push -f it, unless you beat me to it.